src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt

```diff
@@ -29,6 +30,7 @@ class CrashModule(
             val crashes = AttachmentUtils(crashReportExtensions, crashReader).extractCrashesFromAttachments(issue)
 
             assertContainsNewCrash(crashes, lastRun).bind()
+            uploadDeobfuscatedCrashes(issue, crashes)
```

Comment:
Shouldn't it just take the new crashes here? Not all of them? Otherwise it could upload attachments without end.

Reply:
Ah, no it doesn't do that because of Line 58, I see. Still it'd probably be better if it'd continue to work on the new crashes here instead of all of them, at least to save some performance.